packages/drivers/iframe-driver/src/proxyUtils.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Comlink from "comlink";
+
+export const MakeThinProxy = <T>(obj: T): () => Promise<T> => {
```

Comment:
I doubt we'll want to keep this around longer term but it's here mainly to make things a bit easier to read